source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/OverflowActionLayoutRenderer.java

```diff
@@ -0,0 +1,317 @@
+                                               ICardActionHandler cardActionHandler,
+                                               HostConfig hostConfig,
+                                               RenderArgs renderArgs) throws AdaptiveFallbackException
+    {
+        FeatureRegistration featureRegistration = CardRendererRegistration.getInstance().getFeatureRegistration();
+
+        for (BaseActionElement actionElement : elementVector)
+        {
+            IBaseActionElementRenderer actionRenderer = CardRendererRegistration.getInstance().getActionRenderer(AdaptiveCardObjectModel.ModeToString(Mode.Secondary));
+
+            try
+            {
+                if (actionRenderer == null)
+                {
+                    throw new AdaptiveFallbackException(actionElement);
+                }
+
+                if ((featureRegistration != null) && (!actionElement.MeetsRequirements(featureRegistration)))
+                {
+                    throw new AdaptiveFallbackException(actionElement, featureRegistration);
+                }
+
+                contentLayout.addView(actionRenderer.render(renderedCard, context, fragmentManager, actionButtonsLayout, actionElement, cardActionHandler, hostConfig, renderArgs));
+            }
+            catch (AdaptiveFallbackException e)
+            {
+                e.printStackTrace();
+            }
+        }
+    }
+
+    /**
+     * Gets color for a given color attribute.
+     *
+     * @param context        context.
+     * @param colorAttribute attribute name.
+     * @return Id of a color resource or -1 id resource not found.
+     */
+    @ColorInt
+    public static int getAttributeColor(Context context, @AttrRes int colorAttribute)
+    {
+        int[] attrs = {colorAttribute};
+        TypedArray typedArray = context.obtainStyledAttributes(attrs);
+        @ColorInt int color = ContextCompat.getColor(context, typedArray.getResourceId(0, -1));
+        typedArray.recycle();
+        return color;
+    }
+
+    private static OverflowActionLayoutRenderer s_instance = null;
+
+    /**
+     * Responsible for default Overflow rendering behavior.
+     */
+    public static class OverflowActionRenderer implements IOverflowActionRenderer {
+
+        private static OverflowActionRenderer m_innerInstance = null;
+
+        public static OverflowActionRenderer getInstance()
+        {
+            if (m_innerInstance == null)
+            {
+                m_innerInstance = new OverflowActionRenderer();
+            }
+
+            return m_innerInstance;
+        }
+
+
+        @Override
+        public boolean onRenderOverflowAction(@NonNull BaseActionElementVector actionElements, @NonNull ViewGroup viewGroup, boolean isRootLevelActions)
+        {
+            return false;
+        }
+
+        @Override
+        public boolean onDisplayOverflowActionMenu(@NonNull BaseActionElementVector actionElements, @NonNull View view)
+        {
+            return false;
```

Comment:
Same comment as above, but for rendering the overflow menu.

Reply:
Added comment above, which applied to this as well. Implementation not required in the SDK as the default implementation will be always showing `BottomSheetDialog`. 
If clients want to change this behavior for any `Overflow` button, then they can use this callback. So, they can have combination of both.